changelogs/fragments/1264-dict_kv-new-filter.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- dict_kv - Add new filter dict_kv which returns a single key-value pair from two arguments. Useful for generating complex dictionaries without using loops.
```

```suggestion
- Add new filter plugin ``dict_kv`` which returns a single key-value pair from two arguments. Useful for generating complex dictionaries without using loops (https://github.com/ansible-collections/community.general/pull/1264).
```
Also, the fragment should contain a small example (like here: https://github.com/ansible-collections/community.general/pull/359/files#diff-5aca891e85acddb3d38422f89fd36e15599e73cf6661fa8c0883fdc4410bc0aaR2) since there is no other place where users can find any documentation about the filter. (Nobody looks at the source code.)